bin/www

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env node
```

Comment:
The server file is called `www`, it is somewhere in bin folder and imports `app` that is 2 levels up from it. [Useful information here](https://i.ytimg.com/vi/jPLKXEUm0jE/maxresdefault.jpg)